.github/workflows/ci.yml

```diff
@@ -28,8 +21,25 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --workspace --all-targets --features "reqwest-tracing/deprecated_attributes, ${{ matrix.otel_version }}"
+          args: --workspace --all-targets --all-features
```

Comment:
we seem to be already running for `all-features` here, why do we have `test-features` block again below? i am sure i am missing something obvious, please let me know.

Same for clippy 🙈

Reply:
It's possible for clippy to pass and tests to run with all features enabled but not to run with other combinations of features. The `test-features` section covers cases such as when no features are enabled, or when only a few features are enabled

Reply:
amazing, thanks for clarifying TIL